src/lib/Microsoft.Fx.Portability.MetadataReader/DependencyFinderEngineHelper.cs

```diff
@@ -38,6 +40,61 @@ public DependencyFinderEngineHelper(IDependencyFilter assemblyFilter, MetadataRe
             _currentAssemblyName = _reader.GetString(assemblyDefinition.Name);
         }
 
+        private IList<AssemblyReferenceInformation> ComputeAssemblyReferences(MetadataReader metadataReader)
+        {
+            List<AssemblyReferenceInformation> refs = new List<AssemblyReferenceInformation>();
+            foreach (var handle in _reader.AssemblyReferences)
+            {
+                try
+                {
+                    var entry = _reader.GetAssemblyReference(handle);
+
+                    string name = metadataReader.GetString(entry.Name);
+                    string culture = entry.Culture.IsNil ? "neutral" : metadataReader.GetString(entry.Culture);
+                    Version version = entry.Version;
+                    string pkt = FormatPublicKeyToken(metadataReader, entry.PublicKeyOrToken);
+
+                    refs.Add(new AssemblyReferenceInformation(name, version, culture, pkt));
+                }
+                catch (BadImageFormatException)
+                {
+                }
+            }
+            return refs;
+        }
+
+        private static string FormatPublicKeyToken(MetadataReader metadataReader, BlobHandle handle)
```

Comment:
We have extension methods that provide this: [MetadataReaderExtensions.FormatAssemblyInfo](https://github.com/Microsoft/dotnet-apiport/blob/master/src/lib/Microsoft.Fx.Portability.MetadataReader/MetadataReaderExtensions.cs#L71-L83)

Reply:
Sweet!

Reply:
Can you remove this?